doc/ubuntu_release_process.md

```diff
@@ -120,36 +120,28 @@ This is generally not the mechanism that is preferred for any release supported
 #### Cherry-pick Process
 The tool for doing this is in ``uss-tableflip/scripts/cherry-pick``.  It takes as import a commit-ish that it will create a cherry-pick from.
 
-    ## just tag current so you have a revert point.
-    $ git tag --delete xx-current >/dev/null 2>&1; git tag xx-current
-    $ git checkout ubuntu/xenial
-    $ cherry-pick dc2bd79949
-
-The tool will:
+The `cherry-pick` tool will:
 
   * add a new file to debian/patches/ named cpick-<commit>-topic that is dep-8 compatible
   * add that patch to debian/patches/series
   * verify that the patch applies (quilt push -a)
   * give you a chance to edit formatted the debian/changelog entry
   * prompt you to commit the change.
 
-From here you follow along with the snapshot upload process from
-[`dch --release` and beyond](#upstream-snapshot-process).
-
-**Note**: After doing uploading, in order to keep the daily builds working, we will then have to revert change.  This is because the recipe will try to build from trunk, and will fail to apply your cherry-picked patch.  This makes sense... it will grab trunk and then try to apply patches, but the cherry-picked patch will already exist.
-
-To do this:
-
-    $ git checkout ubuntu/xenial  # or whatever release.
-    $ new-upstream-snapshot -v --update-patches-only
-    $ git push upstream HEAD
-
-That will produce 1 or 2 commits on the branch with summary like:
-
-  * refresh patches against master commit 2d6e4219
-  * drop cherry picks included in master commit 2d6e4219
-
-After doing that you can go to the recipe pages (see below) and click
+1. Optionally tag current point so you have revert point
+    ```
+    ## just tag current so you have a revert point.
+    $ git tag --delete xx-current >/dev/null 2>&1; git tag xx-current
+    ```
+1. Cherry pick UPSTREAM_COMMITISH
+   ```
+   $ git featch upstream
+   $ git checkout upstream/ubuntu/xenial -b ubuntu/xenial
+   $ cherry-pick dc2bd79949
+   ```
+1. From here you follow along with the snapshot upload process from [`dch --release` and beyond](#upstream-snapshot-process).
+1. Once published, fix daily builds by [reverting your cherry-pick'd patch](#when-the-daily-recipe-build-fails)
```

Comment:
Should these steps be in this section and not in "When the daily recipe build fails" below?

Reply:
Oh, no, this looks like the right place, but you appear to be dropping the `**Note**` that explains why we're doing this?